xtask-wasm-run-example/src/lib.rs

```diff
@@ -74,16 +79,44 @@ impl RunExample {
             };
         }
 
-        Ok(RunExample { index })
+        Ok(RunExample {
+            index,
+            static_dir,
+            app_name,
+        })
     }
 
     fn generate(self, item: syn::ItemFn) -> syn::Result<proc_macro2::TokenStream> {
         let fn_block = item.block;
-        let index = if let Some(expr) = self.index {
-            let span = expr.span();
-            quote_spanned! {span=> #expr }
```

Comment:
You need to use the `quote_spanned!()` whenever possible otherwise the error messages are less useful for the user